packages/pwa-kit-runtime/package.json

```diff
@@ -40,6 +40,7 @@
     "http-proxy-middleware": "^2.0.6",
     "merge-descriptors": "^1.0.1",
     "morgan": "^1.10.0",
+    "path-to-regexp": "^6.2.1",
```

Comment:
I saw this version is approved in 3pp so it should be okay to use

<img width="552" height="73" alt="Screenshot 2025-08-19 at 5 58 02 PM" src="https://github.com/user-attachments/assets/9260b1fb-af91-4923-bbfd-6d1e55b48508" />